app/api/groups/get_managers.go

```diff
@@ -27,6 +27,8 @@ type GroupManagersViewResponseRowThroughAncestorGroups struct {
 	CanGrantGroupAccessThroughAncestorGroups bool `json:"can_grant_group_access_through_ancestor_groups"`
 	// displayed only when include_managers_of_ancestor_groups=1, note that the group is an ancestor of itself
 	CanWatchMembersThroughAncestorGroups bool `json:"can_watch_members_through_ancestor_groups"`
+	// displayed only when include_managers_of_ancestor_groups=1
```

Comment:
May be worth a more precise explanation so that we do not wonder what "direct" means exactly later.
I would have put the field in every case (always true if `include_managers_of_ancestor_groups`=0) but that's ok like that as well.

Reply:
reworked everything

Reply:
removed the field is_direct_manager